### ISSUE TITLE
JAWS is not announced the aria-checked state

### ISSUE THREAD

**Issue by novsstation** [FreedomScientific/VFO-standards-support#474]
### Summary

JAWS is not announced aria-checked state
Currently accessibility standard defines [aria-checked attribute](https://www.w3.org/TR/wai-aria/#aria-checked)
"The aria-checked attribute indicates whether the element is checked (true), unchecked (false), or represents a group of other elements that have a mixture of checked and unchecked values (mixed)."

But JAWS is not announced it.

### Example:
1. Open the https://codepen.io/SNovikov/pen/VwKwxdm?editors=1111
2. click any times on the "Sample text"

### Expected result

Jaws is announced the changes of the selected attribute

### Actual result

Jaws is not announced the changes of the selected attribute

### Example
https://codepen.io/SNovikov/pen/VwKwxdm?editors=1111

#### JAWS version and build number
 JAWS 2021.2011.16 - November 2020

 
#### Operating System and version
Windows 10

#### Browser and version:
Latest versions of chrome/opera

**Comment by JAWS-test** [comment]
I am unsure if this is a JAWS error because if you add the tabindex=0 to the checkbox, the output is correct.  JAWS does not output your example correctly because your source code is incorrect. However, even if your source code is incorrect, JAWS could output the checkbox correctly...

**Comment by novsstation** [comment]
Thank you for your clarification.

Yes, it works fine with tabindex attribute.

But there is another problem. JAWS is not announced the changes of the selected aria attribute in caes of using the [activedescendant](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-activedescendant_attribute). It successfully supports keyboard navigation, but it doesn't recognize the changes of the aria-changes state.

Please check my modified sample https://codepen.io/SNovikov/pen/VwKwxdm

**Comment by JAWS-test** [comment]
Please use aria-activedescendant according to [specification](https://www.w3.org/TR/wai-aria-1.2/#aria-activedescendant). Your example is invalid and will not work with JAWS. aria-activedescendant is not suitable for checkboxes

**Comment by frex65** [comment]
Since JAWS does announce the state change of both HTML and ARIA checkboxes when correctly used,I will close this issue.
